geode/dispatcher.py

```diff
@@ -266,12 +277,16 @@ class Dispatcher:
     providers: Dict[str, Any] = {}
     dispatcher = None
 
-    def __init__(self, config=None):
+    def __init__(self, config=None, threaded=True):
+        self.threaded = threaded
```

Comment:
What are we doing that we need to thread suddenly?

Reply:
it has been threaded, nothing new added here.
the reason is you can't start event loops when one already exists.
so in any repl, jupyter other env that uses event loops, you need to start a new thread.